src/test/ui/mismatched_types/issue-36053-2.rs

```diff
@@ -1,3 +1,7 @@
+// FIXME: missing sysroot spans (#53081)
+// ignore-i586-unknown-linux-gnu
+// ignore-i586-unknown-linux-musl
+// ignore-i686-unknown-linux-musl
```

Comment:
This seems unfortunate.

Reply:
At least we're tracking the underlying issue now :o)